src/app/component/comments/components/comment-pagination/comment-pagination.component.html

```diff
@@ -4,7 +4,8 @@
                        [maxSize]="calcPaginationSize(p.getLastPage(), p.getCurrent())"
                        (pageChange)="onPageChange($event)"
                        >
-    <div class="custom-pagination">
+    <div class="custom-pagination"
+         *ngIf='p.getTotalItems() > 10'>
```

Comment:
Magic number 10, You should use a variable for that value.

Reply:
It's my mistake. I've fixed it  in this PR https://github.com/ita-social-projects/GreenCityClient/pull/685.